proxyd/cache.go

```diff
@@ -130,10 +130,8 @@ func newRPCCache(cache Cache) RPCCache {
 		"eth_getBlockTransactionCountByHash":    staticHandler,
 		"eth_getUncleCountByBlockHash":          staticHandler,
 		"eth_getBlockByHash":                    staticHandler,
-		"eth_getTransactionByHash":              staticHandler,
```

Comment:
I think we can retain this. A transaction hash maps to the same data regardless of reorgs.

Reply:
the response contains a 
```
transactionIndex: QUANTITY - integer of the transactions index position in the block. null when its pending.
```